docs/source/Getting-Started/Running-Keystone-with-FireSim.rst

```diff
@@ -48,42 +48,75 @@ Checkout ``firesim-1.4.0-keystone`` branch and update submodules.
   git submodule update --init --recursive
 
 
-Build Boot Image
-########################
-
-First, we need to build the Linux kernel with built-in Keystone module, and the Berkeley Bootloader (bbl) containing the Keystone security monitor.
-This command will compile both ``riscv-pk`` and ``riscv-linux``, and create a bootable image. It also build ``buildroot`` to get a disk image.
-
-::
-   
-  ./sw-manager.py build
-
 Build Keystone SDK
 ##########################
 
-Now, you're ready for launching Keystone.
-We provide sample enclaves with Keystone SDK, so let's build the enclaves and copy them to the disk image.
+We provide sample enclaves with Keystone SDK, so let's build the enclaves and copy them to the buildroot.
 Build the Keystone SDK by running following commands:
 
 ::
 
   cd sdk
-  make
+  make all
+  export KEYSTONE_SDK_DIR=$(pwd)
+  
+
+Next, we will copy the binaries into the disk image from the previous part. First, copy 
+tests from the top-level keystone directory:
+
+::
+
+  rm -rf test
```

Comment:
Rather than copying the top-level tests, I'd just add the entire directory to firesim-software.
This will enable immediately bootstraping Keystone on FireSim, without cloning the top-level keystone repo.